cppcheck for incrementTime.c is giving an error

```txt
Checking heclib_c/src/Utilities/DateTime/incrementTime.c ...
heclib_c/src/Utilities/DateTime/incrementTime.c:223:31: error: Invalid abs() argument nr 1. A non-boolean value is required. [invalidFunctionArgBool]
 else if (abs(intervalMinutes > MINS_IN_1_YEAR) <= MINS_IN_1_DAY) {
```

https://github.com/HydrologicEngineeringCenter/hec-dss/blob/main/heclib/heclib_c/src/Utilities/DateTime/incrementTime.c#L223

--- a/heclib/heclib_c/src/Utilities/DateTime/incrementTime.c
+++ b/heclib/heclib_c/src/Utilities/DateTime/incrementTime.c
@@ -214,13 +214,13 @@ int incrementTime(int intervalSeconds, int numberPeriods,
 			day = 15;
 		}
 	}
-	else if (abs(intervalMinutes - MINS_IN_1_MONTH) <= 2 * MINS_IN_1_DAY) {
+	else if (llabs(intervalMinutes - MINS_IN_1_MONTH) <= 2 * MINS_IN_1_DAY) {
 		//  Monthly
 		month += numberPeriods;
 		//  Day is always end of month for standard
 		day = 28;
 	}
-	else if (abs(intervalMinutes > MINS_IN_1_YEAR) <= MINS_IN_1_DAY) {
+	else if (llabs(intervalMinutes - MINS_IN_1_YEAR) <= MINS_IN_1_DAY) {
 		//  Yearly
 		year += numberPeriods;
 		//  Always new year's eve for standard

